doc/dev/role/07_interfaces.md

```
@@ -0,0 +1,185 @@
+イントロ画面で別のチームとして表示させたい場合に実装します。
+- `ExtremeRoleType FakeTeam { get; }`: 表示上のチーム（Crewmate, Impostorなど）。
+
+#### `IRoleSpecialReset`
+役職が剥奪されたり、特殊なリセットが必要な場合に実装します。
+- `void AllReset(PlayerControl rolePlayer)`
+
+---
+
+## 能力・ロジック (AbilityHandler) 向け
+
+#### `IKilledFrom`
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)